### PR TITLE
[AC][SUPP-627] tax regime intersitial page

### PR DESCRIPTION
**Pull request by AndyHWChung**

### CODE REVIEW

File: project/SbtUpdatesSettings.scala [hunk 1]
```diff
@@ -11,7 +11,7 @@ object SbtUpdatesSettings {
     dependencyUpdatesFilter -= moduleFilter("org.scala-lang"),
     dependencyUpdatesFilter -= moduleFilter("com.typesafe.play"),
     // ignore for now until work for VAT starts in this service
-    // dependencyUpdatesFilter -= moduleFilter("uk.gov.hmrc", "essttp-backend-cor-journey"),
+    dependencyUpdatesFilter -= moduleFilter("uk.gov.hmrc", "essttp-backend-cor-journey"),
```
Comment (AndyHWChung) [inline project/SbtUpdatesSettings.scala:14]: to be commented out again in another PR

**Review by barnesjake — APPROVED** [review]
